docs/internals/deprecation.txt

```diff
@@ -105,6 +105,10 @@ details on these changes.
 
 * The ``django.contrib.auth.hashers.CryptPasswordHasher`` will be removed.
 
+* The ``django/forms/default.html`` template will be removed.
```

Comment:
Nit.  Elsewhere  we've got these in quotation marks. I don't think I've got it right in my head if they should be the same.